packages/editor/src/store/selectors.native.js

```diff
@@ -66,3 +66,14 @@ export const isEditedPostAutosaveable = createRegistrySelector(
 export function getClipboard( state ) {
 	return state.clipboard;
 }
+
+/**
+ * Returns the current notice data.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {Object} Current notice data.
+ */
+export function getNotices( state ) {
+	return state.notices?.notices;
```

Comment:
This is totally optional but if we see this is a bit redundant to have `state.notices?.notices` here https://github.com/WordPress/gutenberg/pull/23113/files#diff-72860342a480cb0d0137a1570ca7d6bcR81 we can work with the state directly without `notices:` 

Like:

```
export function notices( state = [], action ) {
	switch ( action.type ) {
		case 'CREATE_NOTICE':
			return [ ...state, action.notice ];
		case 'REMOVE_ALL_NOTICES':
			return [];
		case 'REMOVE_NOTICE':
			return state.filter( ( notice ) => notice.id !== action.id );
	}
	return state;
}
```

And then this would just be:

```suggestion
	return state.notices;
```

But like I said, totally optional, it would work just the same =)

Reply:
Done, thanks! I had no idea why I had an assumption like it shouldn't be an array.